src/archive.rs

```diff
@@ -36,4 +39,57 @@ pub struct Header {
     pub terminator: [u8; 2],
 }
 
+/// The header at the start of an AIX big archive member, without name.
+#[derive(Debug, Clone, Copy)]
+#[repr(C)]
+pub struct AixHeader {
+    /// Member size in decimal.
+    pub size: [u8; 20],
+    /// Offset of next member in decimal.
+    pub next_member: [u8; 20],
+    /// Offset of previous member in decimal.
+    pub prev_member: [u8; 20],
+    /// File modification timestamp in decimal.
+    pub date: [u8; 12],
+    /// User ID in decimal.
+    pub uid: [u8; 12],
+    /// Group ID in decimal.
+    pub gid: [u8; 12],
+    /// File mode in octal.
+    pub mode: [u8; 12],
+    /// Name length in decimal.
+    pub name_length: [u8; 4],
+}
+
+/// Discriminated union for multiple type headers
+#[derive(Debug, Clone, Copy)]
+pub enum MemberHeader {
+    /// GNU or BSD style header
+    SystemV(Header),
+    /// AIX style big archive header
+    AixBig(AixHeader),
+}
+
 unsafe_impl_pod!(Header);
+unsafe_impl_pod!(AixHeader);
+
+/// The AIX big archive fixed len header.
+#[derive(Debug, Clone, Copy)]
+#[repr(C)]
+pub struct AIXBigFixedHeader {
```

Comment:
Please use the documented names for the fields here (you can leave off the `fl_`).

Also please use the documented comments for the fields, since I think they are a bit clearer.